src/chapters/databases/db-practice.rst

```diff
@@ -0,0 +1,171 @@
+#. We indicate those values with arguments inside the ``.execute()`` method.
+#. Note that we do not need to include a *primary key* value when we add a row.
+   This process is automatic with SQLite.
+#. Use the ``.commit()`` method to finish the process.
+
+**Final code from the video**:
+
+.. sourcecode:: Python
+   :linenos:
+
+   import sqlite3
+
+   # Connect to the practice.db database:
+   database = sqlite3.connect('practice.db')
+   cursor = database.cursor()
+
+   # Create the 'students' table if not already there:
+   sql_query = "CREATE TABLE IF NOT EXISTS students (last_name TEXT, first_name TEXT, grad_year INT)"
+   cursor.execute(sql_query)
+
+   # Collect values for the 3 table columns:
+   last_name = input("Enter your last name: ").capitalize()
+   first_name = input("Enter your first name: ").capitalize()
+   grad_year = int(input("Enter your graduation year: "))
+
+   # Add a new row to the 'students' table, them commit.
+   sql_query = "INSERT INTO students (last_name, first_name, grad_year) VALUES (?, ?, ?)"
+   cursor.execute(sql_query, (last_name, first_name, grad_year))
+   database.commit()
+
+.. _sqlite-explorer:
+
+Viewing a Table in VS Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To view the data stored in ``practice.db``, we need to add an extension to
+Visual Studio Code. Follow along with this video to make the update:
+
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube.com/embed/0rlBCHWAemk" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
+**Summary**:
+
+#. Click the extensions button in VS Code. Enter ``SQLite`` in the search bar,
+   then select and install the extension by ``alexcvzz``.
+
+   .. figure:: figures/sqlite-extension.png
+      :alt: The extensions tab, search bar, and extension info for SQLite by alexcvzz.
+      :width: 70%
+
+      It only takes about 3 clicks to install the SQLite extension.
+
+#. Return to the file tree. Right-click on ``practice.db`` and select
+   *Open Database*. At the bottom-left of the VS Code workspace, open the
+   *SQLITE EXPLORER* tab. Expand ``practice.db``, then click on the *Play*
+   button next to the ``students`` table.
+
+   .. figure:: figures/view-table.png
+      :alt: Showing SQLite Explorer tab, with Play button next to students table.
+      :width: 50%
+
+      Click the Play arrow to view the contents of the student table.
+
+Retrieve Data From Table
+------------------------
+
+Code along with this video as we access the information in the database and
+display it in the console.
+
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube.com/embed/CPHPjP-CRnA" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
```

Comment:
In the video you can see you commented out the code from the section above, but students might miss it.  Thinking a note above this section might work, so as to not have to reshoot the video.

Reply:
Got it.  I added text to indicate which lines the students should comment out.